ironfish-cli/bin/run

```diff
@@ -24,7 +24,4 @@ if (Number(process.versions.node.split('.')[0]) < MIN_NODE_VERSION) {
   process.exit(1)
 }
 
-require('@oclif/core')
-  .run()
-  .then(require('@oclif/core/flush'))
-  .catch(require('@oclif/core/handle'))
+require('@oclif/core').execute({ dir: __dirname })
```

Comment:
This change makes me a *little* nervous, but is the new? default according to the docs. Doing it the way we had was making it throw stack traces for even non-error things which was very strange.